zipkin-common/src/main/scala/com/twitter/zipkin/common/Trace.scala

```diff
@@ -0,0 +1,17 @@
+package com.twitter.zipkin.common
+
+/** Utilities for working on traces */
+object Trace {
+
+  /** What's the total duration of the spans in this trace? */
+  def duration(spans: List[Span]): Option[Long] = {
```

Comment:
@yurishkuro this is the best I could do with scala.

Feel free to offer another solution post-merge

I'm suggesting post-merge because my experience with people's suggestions on scala is that they don't work. After this is merged, we have tests that will pass or fail based on an optimized solution as opposed to what I'm usually subjected to, which is "try this!" where "this" doesn't work for some reason.